CI/Dockerfile

```
@@ -33,63 +61,212 @@ RUN apt-get -y update; \
+ARG HDF5_VERSION
+ARG install_dir
+ARG build_dir
+ARG ci_jobs
+ARG CXX
+ARG CC
+
 ENV hdf5_build_dir=${build_dir}/hdf5
 ENV hdf5_install_dir=${install_dir}/hdf5
 
-# Build HDF5
-# HDF5 argument possible value: 1.10.4 or system
-ARG HDF5=1.10.4
-ENV HDF5_VERSION=${HDF5}
-#TODO move sh file contents into this Dockerfile
-RUN /root/etc/CI/docker/build_hdf5.sh
+RUN mkdir -p ${hdf5_build_dir}/build && \
+    cd ${hdf5_build_dir} && \
+    wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-${HDF5_VERSION_major}/hdf5-${HDF5_VERSION}/src/hdf5-${HDF5_VERSION}.tar.gz && \
+    tar -xzf hdf5-${HDF5_VERSION}.tar.gz && \
+    cd build && \
+    ../hdf5-${HDF5_VERSION}/configure --enable-shared \
+        --prefix=${hdf5_install_dir} \
+        CXX=${CXX} \
+        CC=${CC} && \
+    make -j${ci_jobs} && \
+    make install && \
+    cd && \
+    rm -rf ${hdf5_build_dir}
+
 
 FROM hdf5 AS moab
 
-ENV PYTHONPATH=/root/build_dir/moab/bld/pymoab/lib/python3.8/site-packages
+# accessing gloabl ARGs in build stage
+ARG install_dir
+ARG MOAB_BRANCH
+ARG build_dir
+ARG ci_jobs
+ARG CXX
+ARG CC
+
 
 # Set MOAB env variable
 ENV moab_build_dir=${build_dir}/moab
 ENV moab_install_dir=${install_dir}/moab
 
-ARG MOAB=5.3.0
-ENV MOAB_VERSION ${MOAB}
-#TODO move sh file contents into this Dockerfile
-RUN if [ "${MOAB_VERSION}" != "master" ] && [ "${MOAB_VERSION}" != "develop" ]; then \
-        /root/etc/CI/docker/build_moab.sh; \
-    fi;
-    
\ No newline at end of file
+
+RUN mkdir -p ${moab_build_dir}/build && \
+    cd ${moab_build_dir} && \
+    git clone -b ${MOAB_BRANCH} --depth 1 https://bitbucket.org/fathomteam/moab && \
+    cd build && \
+    # static building of moab is also an option and was required for older versions
+    # cmake ../moab -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+    #               -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+    #               -DCMAKE_CXX_COMPILER=${CXX} \
+    #               -DCMAKE_C_COMPILER=${CC} \
+    #               -DENABLE_BLASLAPACK=OFF \
+    #               -DBUILD_SHARED_LIBS=OFF \
+    #               -DENABLE_FORTRAN=OFF && \
+    # make -j${ci_jobs} && \
+    # make install && \
+    # rm -rf * && \
+    cmake ../moab -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
+                  -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+                  -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+                  -DCMAKE_CXX_COMPILER=${CXX} \
+                  -DCMAKE_C_COMPILER=${CC} \
+                  -DENABLE_BLASLAPACK=OFF \
+                  -DBUILD_SHARED_LIBS=ON \
+                  -DENABLE_FORTRAN=OFF \
+                  -DENABLE_PYMOAB=ON && \
+    mkdir -p ${moab_build_dir}/build/pymoab/lib/python3.6/site-packages && \
+    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make -j${ci_jobs} && \
+    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make install && \
```

remove any reference to this hardcoded python3.6 site-packages path
```suggestion
    make -j${ci_jobs} && \
    make install && \
```

---

these lines have now been removed so I think this is resolved